contracts/src/Gauge.sol

```diff
@@ -12,7 +12,9 @@ import "openzeppelin/contracts/utils/Strings.sol";
 import "base64/base64.sol";
 import "hot-chain-svg/SVG.sol";
 import "./interfaces/IMerc.sol";
-import "./PledgedMerc.sol";
+import "./PledgingVault.sol";
+import "./StakingVault.sol";
```

Comment:
We may want to move to interfaces here if this contract gets too beefy.

Reply:
We need to have the actual contracts because we do a `new` in the constructor. We then make `Clones` for each of the gauges.